src/login/10-systemd-logind-root-ignore-inhibitors.rules.example

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: LGPL-2.1-or-later
+//
+// This file is part of systemd.
+//
+// systemd is free software; you can redistribute it and/or modify it
+// under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation; either version 2.1 of the License, or
+// (at your option) any later version.
+
+// Allow the root user to ignore inhibitors when calling reboot etc.
```

Comment:
maybe also suggest how to enable this. (which I guess would work by symlinking this from /usr/share into the matching dir in /etc/? assuming that polkit is smart enough to allow that)